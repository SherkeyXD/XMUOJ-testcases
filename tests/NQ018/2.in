86
78 86 48 80
35 72 10 44
90 28 45 63
10 8 25 20
44 74 20 92
86 14 75 32
20 67 88 33
40 55 94 92
40 75 20 78
46 2 20 50
81 76 40 53
70 32 52 5
80 16 28 6
80 50 97 14
52 64 10 62
75 45 75 90
81 82 23 8
35 86 80 56
59 20 91 80
86 59 35 20
68 76 13 76
8 12 90 78
56 45 75 20
36 54 99 10
55 64 14 36
71 68 58 92
50 50 60 30
98 56 70 96
16 37 40 32
58 88 44 72
24 70 16 80
35 46 10 60
36 32 41 40
19 78 72 50
44 76 70 48
40 63 78 84
14 60 55 88
11 31 75 57
20 72 63 92
30 71 62 36
49 79 96 92
27 60 76 20
38 3 34 70
97 4 60 98
78 49 52 69
12 83 60 30
76 56 45 68
2 50 74 74
56 72 62 27
30 74 36 50
43 55 37 90
88 15 86 88
71 65 52 43
24 44 92 87
20 40 25 21
72 40 24 36
68 80 12 26
8 73 85 10
40 13 45 86
80 90 51 28
36 5 46 50
60 35 82 59
13 69 31 25
17 12 72 12
65 34 30 36
6 34 24 96
74 70 66 16
60 10 92 28
26 77 66 77
82 64 12 15
77 20 56 80
85 17 65 80
24 28 9 45
50 56 16 60
16 90 94 8
40 21 75 95
54 44 28 95
40 76 98 90
76 96 56 98
33 62 40 68
36 70 10 25
23 64 50 89
78 64 61 64
99 99 78 93
57 52 80 40
80 78 26 49
