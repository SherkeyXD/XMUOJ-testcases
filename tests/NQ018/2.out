14883.227
12065.484
3252.707
2407.861
1981.322
2028.701
10073.502
36174.995
5137.159
3619.039
22640.237
10112.437
11428.443
6438.235
11726.743
18105.525
36204.161
5504.847
32507.249
25464.501
18162.123
39360.768
1632.013
5422.836
16500.490
4469.635
2665.811
5382.510
3379.317
11114.345
524.677
2376.612
2737.241
7800.135
634.985
22123.330
21184.329
22173.657
22022.933
796.915
26837.599
2392.160
12868.801
20785.483
2136.885
1236.739
6613.597
27187.026
11932.491
4815.431
9198.798
27760.398
13391.959
38791.342
2081.769
12679.949
28936.464
3583.645
20248.870
32021.466
10631.501
15351.358
5030.425
8828.416
7726.627
18071.015
19783.533
13445.024
13944.204
29129.562
12793.743
14961.735
277.193
7748.535
1063.495
35581.087
7759.123
30985.505
8314.181
3732.839
14144.453
16621.335
6431.159
14100.442
3587.170
27367.285
